source/common/secret/sds_api.cc

```diff
@@ -99,7 +95,7 @@ void CertificateValidationContextSdsApi::updateConfigHelper(
   if (new_hash != secret_hash_ &&
       secret.type_case() == envoy::api::v2::auth::Secret::TypeCase::kValidationContext) {
     secret_hash_ = new_hash;
-    secrets_ =
+    certificate_validation_context_secrets_ =
```

Comment:
these two updateConfigHelper are almost the same.  can we move them to the sds_api base class

Reply:
Done.